modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -0,0 +1,111 @@
+[id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
+= Configuration of JSON log data for default Elasticsearch
+
+With the default Elasticsearch log store, you must create an index for each format if the JSON log entries _have different formats_.
```

Comment:
"With the default Elasticsearch log store" - I'm not sure why my eyes stopped here. Maybe it wasn't clear 100% what's going on with the log store? Maybe adding a gerund will add a bit of clarity? E.g. "When using X" or " When working with"? Just nitpicking perhaps.

Reply:
Updated. Thanks.